backend/src/main/java/capstone/myfinancemanager/manager/service/TransactionServiceImp.java

```diff
@@ -0,0 +1,33 @@
+package capstone.myfinancemanager.manager.service;
+
+
+import capstone.myfinancemanager.manager.model.dto.TransactionDto;
+import capstone.myfinancemanager.manager.respository.TransactionRepo;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
+@AllArgsConstructor
+@Service
+public class TransactionServiceImp {
```

Comment:
```suggestion
public class TransactionService {
```

Impl an die konkreten Klassen zu schreiben, finde ich nicht so schön. Hat man früher viel gemacht aber mittlerweile kaum noch